pkg/kubelet/kubelet.go

```diff
@@ -1623,6 +1629,13 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 				return
 			}
 			kl.podManager.UpdatePods(u, podSyncTypes)
+			if u.Op == cache.ADD {
+				// TODO: Break out SyncPods and check admission etc
+				for _, p := range u.Pods {
+					kl.podWorkers.UpdatePod(p, nil, func() {})
+				}
+				continue
+			}
 			unsyncedPod = true
 		case <-time.After(kl.resyncInterval):
```

Comment:
That's what i intend to do, not what this pr does :)
So today we sync the whole world on creation. If there are no creations, we sync every 10s. I'm dropping it so we sync every 10s and fastpath the creates.